### ISSUE TITLE
[Bug] 当使用internvl3代码，放开全参数训练的时候，loss会在两三个step后变成0

### ISSUE THREAD

**Issue by Wangman1** [OpenGVLab/InternVL#1043]
### Checklist

- [ ] 1. I have searched related issues but cannot get the expected help.
- [ ] 2. The bug has not been fixed in the latest version.
- [ ] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.

### Describe the bug

使用internvl3进行sft的时候，当全参数都放开训练的时候，loss 训练两三个step后就变成0 了，当 freezevit 时（放开llm和mlp）的时候，loss就不会变为0，如下图所示

<img width="1020" alt="Image" src="https://github.com/user-attachments/assets/49aafb0c-6f15-4f9b-8320-84be9355c749" />

### Reproduction

训练命令如下：

torchrun $DISTRIBUTED_ARGS internvl/train/internvl_chat_finetune.py \
  --model_name_or_path "Internvl1b" \
  --conv_style "Hermes-2" \
  --output_dir ${OUTPUT_DIR} \
  --meta_path "train_data.json" \
  --overwrite_output_dir True \
  --force_image_size 448 \
  --max_dynamic_patch 12 \
  --down_sample_ratio 0.5 \
  --drop_path_rate 0.1 \
  --freeze_llm False \
  --freeze_mlp False \
  --freeze_backbone False \
  --vision_select_layer -1 \
  --dataloader_num_workers 2 \
  --bf16 True \
  --num_train_epochs 4 \
  --per_device_train_batch_size ${PER_DEVICE_BATCH_SIZE} \
  --gradient_accumulation_steps ${GRADIENT_ACC} \
  --evaluation_strategy "no" \
  --save_strategy "steps" \
  --save_steps 1000 \
  --save_total_limit 1 \
  --learning_rate 4e-5 \
  --weight_decay 0.01 \
  --warmup_ratio 0.03 \
  --lr_scheduler_type "cosine" \
  --logging_steps 1 \
  --max_seq_length 8192 \
  --do_train True \
  --grad_checkpoint True \
  --group_by_length True \
  --dynamic_image_size True \
  --use_thumbnail True \
  --ps_version 'v2' \
  --deepspeed "zero_stage3_config.json" \
  --report_to "tensorboard" \
  2>&1 | tee "${OUTPUT_DIR}/training_log.txt"

### Environment

```Shell
transformers 4.37.2
```

### Error traceback

```Shell

```

**Comment by XCF-Mike** [comment]
你好，请教一下，现在的训练环境是根据requirements搭建的嘛，因为我看这个文件的更新时间已经很久了

**Comment by blofn** [comment]
我也碰到了一样的问题
